### ISSUE TITLE
FreeBSD Install

### ISSUE THREAD

**Issue by bill-mccormick-dg** [jumpmanjay/HDGrandSlam.bundle#13]
When I try to play a channel, it just spins, plays nothing. Need to reboot to get back the DVR. 

Is there a log I can glean more info from? How does the debug script work?

**Comment by bill-mccormick-dg** [comment]
More info: Live TV actually does work in one case ... on a Rasplex client running OpenPHT. No success on Windows (tried Firefox and Chrome) and Android Plex client.

It would be nice we could improve the WAF (Wife Acceptance Factor), as having to navigate through the the Beta DVR Program Guide to get a listing, and then deep through the HDGrandSlam Channel/LiveTV tree isn't going to play.

**Comment by bill-mccormick-dg** [comment]
Also works on an iPAD Plex client

**Comment by nicktones** [comment]
Yeah plugins and various bits of plex get their own log. The log folder will be in the same place as the plugins folder.
